Applications/WebPortal/WebApp/library/source/pages/account_page/compliance_page/parameter_entry.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
+import * as Nexus from 'nexus';
+import { DisplaySize } from '../../../display_size';
```

Comment:
Alphabetical order here too, "components" comes before "display_size".

Alternatively you can do this:

    import { CurrencySelectionBox, DisplaySize, ... } from '../../..';